book/scheduling.md

```diff
@@ -1374,37 +1385,15 @@ if __name__ == "__main__":
         browser.schedule_animation_frame()
 ```
 
-Remove the `schedule_animation_frame` call from inside the callback
-inside `schedule_animation_frame`. We're now scheduling an animation
-frame every time raster-and-draw is finished.
-
 And that's it: we should now be doing render on one thread and raster
 and draw on another!
 
 ::: {.further}
-Unfortunately, Python currently has a [global interpreter lock][gil],
-so our two Python threads don't truly run in parallel,[^why-gil]
-and our browser's *throughput* won't increase much from multi-threading.
-Nonetheless, the *responsiveness* of the browser thread is still
-massively improved, since it often isn't blocked on JavaScript or the
-front half of the rendering pipeline. This is an unfortunate
-limitation of Python that doesn't affect real browsers, so try to
-pretend it's not there.[^why-locks]
+The `needs_raster_and_draw` dirty bit doesn't just make the browser a
```

Comment:
As mentioned in the other comment, I think this is a better place for the global lock go-further. We can move this go-further into a footnote.

Reply:
How about adding the other sentences from the GIL go-further here? "The go-further in section X said some stuff; here is some more about the implications?" Anyway just an idea, if the sentences fit.

Reply:
See other thread.